build.sc

```diff
@@ -0,0 +1,56 @@
+// IMPORTANT: the version of the contrib modules must be 0.4.0-9-1db2ee or later.
+// IMPORTANT: if you change this version to a release, please update the .travis.yml
+// FIXME: change to a release version after 0.4.0-9-1db2ee
+import $ivy.`com.lihaoyi::mill-contrib-buildinfo:0.4.0-9-1db2ee`
+import mill.contrib.scoverage.ScoverageModule
+
+import mill.scalalib.{PublishModule, Dep, DepSyntax, ScalaModule}
+import mill.scalalib.publish.{PomSettings, License, Developer, SCM}
+import ammonite.ops._
+
+import scala.sys.process.Process
+
+object commons extends ScalaModule with ScoverageModule with PublishModule {
+  def artifactName = "commons"
```

Comment:
Could we do something like `getClass.getSimpleName` here to standardize object == artifact name?  Or do we want to embed the version name here?